rust/Cargo.toml

```diff
@@ -24,3 +24,7 @@ members = [
         "integration-testing",
 	"benchmarks",
 ]
+default-members= [
```

Comment:
Why add this? is this related to the change?

Reply:
Is this just so that wasm builds by default at the top level?  If so, I'd rather not do this either.  i.e. it's enough to just say that we can build the sub-crates on wasm instead of changing this global default.

Reply:
Yes. This was add as I did not try and port the Flight and Parquet submodules as I believe their core functionality uses file and sockets which don't have stable support in WASM yet. 

I'll try out the individual submodule build, add details to readme and revert the Cargo.toml change.